HW3_Daria_Dolia.py

```diff
@@ -0,0 +1,51 @@
+import random
+
+
+def get_valid_input():
+    while True:
+        try:
```

Comment:
It's better to put inside `try...except` block only the code that can raise an exception